bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/PowerMaxBinding.java

```diff
@@ -0,0 +1,907 @@
+                } else {
+                    comm.requestArmMode(armMode, currentState.isPowerlinkMode()
+                            ? PowerMaxPanelSettings.getThePanelSettings().getFirstPinCode() : pinCode);
+                }
+                break;
+            } else if (actionPGMX10 != null) {
+                comm.sendPGMX10(actionPGMX10, devicePGMX10);
+                break;
+            } else if (bypass != null) {
+                if ((currentState.isPowerlinkMode() == null) || currentState.isPowerlinkMode().equals(Boolean.FALSE)) {
+                    logger.warn("PowerMax alarm binding: Bypass option only supported in Powerlink mode");
+                } else if (!PowerMaxPanelSettings.getThePanelSettings().isBypassEnabled()) {
+                    logger.warn("PowerMax alarm binding: Bypass option not enabled in panel settings");
+                } else {
+                    comm.sendZoneBypass(bypass.booleanValue(), zoneNr,
+                            PowerMaxPanelSettings.getThePanelSettings().getFirstPinCode());
+                }
+                break;
+            } else if (commandStr != null) {
+                if (commandStr.equalsIgnoreCase("get_event_log")) {
+                    comm.requestEventLog(currentState.isPowerlinkMode()
+                            ? PowerMaxPanelSettings.getThePanelSettings().getFirstPinCode() : pinCode);
+                } else if (commandStr.equalsIgnoreCase("download_setup")) {
+                    if ((currentState.isPowerlinkMode() == null)
+                            || currentState.isPowerlinkMode().equals(Boolean.FALSE)) {
+                        logger.warn("PowerMax alarm binding: download setup only supported in Powerlink mode");
+                    } else {
+                        comm.startDownload();
+                        if (currentState.getLastKeepAlive() != null) {
+                            currentState.setLastKeepAlive(System.currentTimeMillis());
+                        }
+                    }
+                } else if (commandStr.equalsIgnoreCase("log_setup")) {
+                    settings.log();
+                } else if (commandStr.equalsIgnoreCase("help_items")) {
+                    settings.helpItems();
+                } else {
+                    logger.warn("PowerMax alarm binding: rejected command {}", commandStr);
+                }
+                break;
+            }
+        }
+    }
+
+    /**
+     * @{inheritDoc
+     */
+    @Override
+    protected void internalReceiveUpdate(String itemName, State newState) {
+        // the code being executed when a state was sent on the openHAB
+        // event bus goes here. This method is only called if one of the
+        // BindingProviders provide a binding for the given 'itemName'.
+        logger.debug("internalReceiveUpdate({},{}) is called", itemName, newState);
+        logger.warn("Item {}: state can be updated only by the PowerMax alarm binding.", itemName);
+    }
```

Comment:
Could you just remove this override altogether, since the binding is not interested in others updating the state of an item it's bound to?  Also, the WARN message isn't accurate "state can be updated only by the PowerMax alarm binding" because it *can* be updated by a rule or another binding -- it's just that your binding doesn't care.